qutebrowser/utils/log.py

```diff
@@ -291,6 +303,20 @@ def _init_formatters(level, color, force_color, json_logging):
     return console_formatter, ram_formatter, html_formatter, use_colorama
 
 
+def change_console_formatter(level):
+    """Change console formatter based on level.
+
+    Args:
+        level: The numeric logging level
+    """
+    if isinstance(console_handler.formatter, ColoredFormatter):
```

Comment:
I can do that, but the name of the function is `change_console_formatter` which should indicate it isn't going to change the JSONFormatter.